include/odp/api/spec/packet_io.h

```diff
@@ -260,6 +260,10 @@ int odp_pktin_queue(odp_pktio_t pktio, odp_pktin_queue_t queues[], int num);
  * odp_queue_enq_multi(). Behaviour is undefined if other events than packets
  * are enqueued. Application cannot dequeue from these queues.
  *
+ * Static references, referencing and referenced packets may be sent without
+ * the dont_free option only if the relevant packet_ref capability is set
```

Comment:
Refer to the full path of `dont_free`?

Reply:
I am not sure what path you mean but added a reference to odp_packet_free_ctrl_set() in v5.